lambdas/journeyengine/src/main/java/uk/gov/di/ipv/core/journeyengine/JourneyEngineHandler.java

```diff
@@ -217,6 +213,18 @@ private JourneyEngineResult executeJourneyEvent(
         }
     }
 
+    private void validateJourneyStep(String journeyStep) throws JourneyEngineException {
+        Arrays.stream(JourneyStep.values())
+                .filter(step -> step.toString().equalsIgnoreCase(journeyStep))
+                .findFirst()
+                .orElseThrow(
```

Comment:
Sonarcloud is upset about not using the return value of `orElseThrow`. I'm a little confused by it...

Reply:
Maybe you need to collect the stream or end it?

Reply:
Fixed!